Vagrant/bootstrap.sh

```diff
@@ -215,7 +276,29 @@ if [ "$app_env" == "production" ]; then
     $RUN_AS_INSTALLUSER RAILS_ENV=production bundle exec rake assets:precompile
     $RUN_AS_INSTALLUSER RAILS_ENV=production bundle exec rake datarepo:setup_defaults
 fi
+# Fix up configuration files
+# 1. FITS
+$RUN_AS_INSTALLUSER sed -i "s@config.fits_path = \".*\"@config.fits_path = \"$fitsdir/$fitsver/fits.sh\"@" config/initializers/sufia.rb
```

Comment:
It wasn't currently causing trouble, but hard-coding it to /home/vagrant/... in the sufia.rb file means it won't work in AWS or in the case where $fitsdir points to somewhere other than /home/vagrant...

This change ensures the value in the sufia.rb config file matches the location used in the installation script.